_static/js/custom.js

```diff
@@ -250,6 +250,16 @@ $(document).ready(() => {
     registerOnScrollEvent(mediaQuery);
   });
 
+  // Add line-break suggestions to code links
+  const referenceInternalList = document.body.querySelectorAll(".reference.internal");
+  for (const el of referenceInternalList) {
+    if (el.innerText.includes(" ") || el.innerText.length < 10) {
+      continue;
+    }
+
+    el.innerHTML = el.innerText.replace(/((?:(?<!^|[A-Z0-9])[A-Z0-9])|(?:(?<!^)[A-Z0-9](?=[a-z])))/gm, "<wbr>$1");
```

Comment:
I'd ignore numbers and consider them a part of the previous substring. Because while we have `2D` and `3D`, we also have just numbers at the end, and those may end up looking bad:

![image](https://user-images.githubusercontent.com/11782833/204640657-46c4b36c-0156-4508-b79d-bbee8210f2e1.png)

Reply:
I could add a check for the end line.

Reply:
That could work, but this regex is already unnerving 🙃